AutoCollection/HttpRequests.ts

```diff
@@ -176,6 +177,9 @@ class AutoCollectHttpRequests {
             return;
         }
 
+        // Register performance counter monitor
+        AutoColllectPerformance.countRequest(request, response);
```

Comment:
It looks like `countRequest()` will just calculate `start` time and then schedule continuation on end request to calculate the overall request duration and assign it to `lastRequestExecutionTime`. I think that we have this time calculated already at the moment we execute `TrackRequest()` later on - can we reuse that time and avoid double calculation of the same/similar value?

Reply:
Great idea! I'll have to make sure that that code is reachable even when automatic request collection is disabled, but if so, that makes total sense